### PR TITLE
[MRG] Fix to plot_joint title.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/4259

Are we happy with this @jona-sassenhagen  @mmagnuski ?

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1272,7 +1273,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     ts_ax.set_title('')
     if title is not None:
         title_ax = plt.subplot(4, 3, 2)
-        title = ', '.join([title, old_title]) if len(title) > 0 else old_title
+        if title == '':
+            title = ', '.join([title,
+                               old_title]) if len(title) > 0 else old_title
```
Comment (mmagnuski) [inline mne/viz/evoked.py]: `if len(title) > 0` would always be false `if title == ''`

**Comment by mmagnuski** [comment]
apart from my comment - looks good.

**Comment by agramfort** [comment]
@mmagnuski  merge if you're happy

**Comment by mmagnuski** [comment]
I am happy, merging 🚀
Thanks @jaeilepp !

**Pull request merged**